src/api/protectedApiClient.ts

```diff
@@ -506,6 +533,32 @@ const addSites = (request: AddSitesRequest): Promise<void> => {
   );
 };
 
+const sendEmail = (request: SendEmailRequest): Promise<void> => {
+  return AppAxiosInstance.post(AdminApiClientRoutes.SEND_EMAIL, request).then(
+    (res) => res.data,
+  );
+};
+
+const filterSites = (
+  treeCommonNames: string[] | null,
+  adoptedStart: Date | null,
+  adoptedEnd: Date | null,
+  lastActivityStart: Date | null,
+  lastActivityEnd: Date | null,
+  neighborhoodIds: number[] | null,
+): Promise<FilterSitesResponse> => {
```

Comment:
Would it better for the return type to be `Promise<{ filteredSites: FilteredSite[] }>`? IMO it feels a little unnecessary to create a new type for an object with one field.

Reply:
I think it's ok because like you mentioned in the comment above, if we were to change its type (like if we wanted to change the name `filteredSites` to something else), we'll only have to change in one place